hermes-common/src/main/java/pl/allegro/tech/hermes/domain/topic/schema/ZookeeperSchemaSourceProvider.java

```diff
@@ -0,0 +1,37 @@
+package pl.allegro.tech.hermes.domain.topic.schema;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.zookeeper.KeeperException;
+import pl.allegro.tech.hermes.api.SchemaSource;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.exception.InternalProcessingException;
+import pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths;
+
+import java.nio.charset.Charset;
+import java.util.Optional;
+
+public class ZookeeperSchemaSourceProvider implements SchemaSourceProvider {
+
+    protected static final String SCHEMA_SUFFIX = "schema";
```

Comment:
i think we should decentralize it a bit more - this schema is an optional element, not everyone has to know where schema is located. topic/group/sub path is crucial and used throughout the app, so it fits into `ZookeeperPaths`